src/nanovg_mtl.m

```diff
@@ -1558,3 +1558,16 @@ void mnvgReadPixels(NVGcontext* ctx, int image, int x, int y, int width,
          fromRegion:MTLRegionMake2D(x, y, width, height)
         mipmapLevel:0];
 }
+
+int mnvgCreateImageFromHandle(NVGcontext* ctx, void* textureId, int imageFlags) {
+  MNVGcontext* mtl = (MNVGcontext*)nvgInternalParams(ctx)->userPtr;
+  MNVGtexture* tex = mtlnvg__allocTexture(mtl);
+  
+  if (tex == NULL) return 0;
+  
+  tex->type = NVG_TEXTURE_RGBA;
+  tex->tex = (id<MTLTexture>) textureId;
```

Comment:
Remove the space in front of `textureId`.

Reply:
done